ckan/lib/jobs.py

```diff
@@ -36,6 +36,7 @@
 log = logging.getLogger(__name__)
 
 DEFAULT_QUEUE_NAME = u'default'
+DEFAULT_JOB_TIMEOUT = config[u'rq.bg_job_timeout']
```

Comment:
This should be:
```python 
config.get('rq.bg_job_timeout', 180) # Or whatever default value is being used now
```

Otherwise you'll get `KeyError` if people haven't add it to their ini files. That's causing your current test failures.

Other than that it looks good and really useful!